Sources/SWBProjectModel/PIFGenerationModel.swift

```diff
@@ -295,6 +295,7 @@ public enum PIF {
             case executable = "com.apple.product-type.tool"
             case hostBuildTool = "com.apple.product-type.tool.host-build"
             case unitTest = "com.apple.product-type.bundle.unit-test"
+            case swiftpmTestRunner = "com.apple.product-type.tool.swiftpm-test-runner"
```

Comment:
Why does this need to be called "SwiftPM" test runner?

Reply:
I called it that to differentiate it from other kinds of test runners like XCTRunner, but I'm not too attached to the name